mirrulations-client/src/mirrclient/saver.py

```diff
@@ -0,0 +1,70 @@
+import os
+from json import dumps, load
+
+
+class Saver:
+    """
+    A class which takes the content of an attachment (pdf, doc, etc.), and
+    saves it to a given path.
+    ...
+    Methods
+    -------
+    save(content = response, path = string)
+        Takes the content (pdf, doc, etc.) and saves the attachment to a
+        given path
+    """
+
+    def make_path(self, path):
+        try:
+            os.makedirs(f'/data{path}')
```

Comment:
The `os.makedirs()` method has an optional argument `exist_ok` that I believe is `False` by default. If we set that to `True`, that should eliminate the need for a `try/except` block. This would mean that it would create all directories that do `not` already exist.

I am doing this already inside of the `Extractor` class.

Reply:
Apart from that, I like the rest.